### PR TITLE
typo removed in uma.kt file and added one more list and task in the s…

### PR DESCRIPTION
**Pull request by Prathamesh1910**
…ame. Also updated the Readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-""
+"kuchh kam nhi tha, free tha to kuchh faltu bana diya"
```
Comment (Cosmicoppai) [inline README.md]: change this line

**Review by Cosmicoppai — CHANGES_REQUESTED** [review]
do the necessary changes

File: README.md (hunk 1 above)
Comment (Prathamesh1910) [reply README.md]: did the necessary changes

**Review by Cosmicoppai — APPROVED** [review]
Good job

**Pull request merged**